packages/cli/src/commands/DebugCommand.ts

```diff
@@ -27,8 +28,15 @@ export class DebugCommand implements CommandModule {
     try {
       const config = await CLIHelper.getConfiguration();
       CLIHelper.dump(` - configuration ${colors.green('found')}`);
-      const tsNode = config.get('tsNode');
 
+      const isConnected = await CLIHelper.isDBConnected();
+      if (isConnected) {
+        CLIHelper.dump(`   - ${colors.green('database connected')}`);
+      } else {
+        CLIHelper.dump(`   - ${colors.yellow('database not found')}`);
```

Comment:
while i do understand the additional indent, i'd rather keep it on the same level as other checks, as they are also about the configuration...

also, lets improve the messages a bit, the cause might be different than db not existing. latest versions also ensure the database existing during the init phase, so unless you dont have some other problem in the config, this should never happen

```suggestion
        CLIHelper.dump(` - ${colors.green('database connection succesful')}`);
      } else {
        CLIHelper.dump(` - ${colors.yellow('database connection failed')}`);
```